projects/batfish/src/main/java/org/batfish/main/Batfish.java

```diff
@@ -2769,6 +2715,41 @@ private ValidateEnvironmentAnswerElement loadValidateEnvironmentAnswerElement(
     }
   }
 
+  private void mergeConvertAnswer(
+      boolean summary, boolean verboseError, InitInfoAnswerElement answerElement) {
+    ConvertConfigurationAnswerElement convertAnswer = loadConvertConfigurationAnswerElement();
+    if (!summary) {
+      if (verboseError) {
+        SortedMap<String, Set<BatfishStackTrace>> errors = answerElement.getErrors();
+        convertAnswer
+            .getErrors()
+            .forEach(
+                (hostname, convertErrors) -> {
+                  errors.computeIfAbsent(hostname, k -> new HashSet<>()).add(convertErrors);
+                });
+      }
+      SortedMap<String, org.batfish.common.Warnings> warnings = answerElement.getWarnings();
+      convertAnswer
+          .getWarnings()
+          .forEach(
+              (hostname, convertWarnings) -> {
+                org.batfish.common.Warnings combined = warnings.get(hostname);
+                if (combined == null) {
+                  warnings.put(hostname, convertWarnings);
```

Comment:
do you want this `put` to clone `convertWarnings`? seems like you might mutate warnings from one source if merging in another source later.